integr8ly/cloud-resource-operator
# Updating operator-sdk to v0.12.0

## Overview
https://issues.jboss.org/browse/INTLY-4216

Unfortunately there is one potential issue in this PR that I haven't been able to get resolved. The CRDs now have a `$ref` field for the spec and status instaed of the full object tree. I have tried everything I know about the generation to try and get this working again but it needs someone with a better understanding of the k8s code-generator (https://github.com/kubernetes/code-generator).

## Verification

- Clone this branch
- Run `make cluster/prepare`
- Run `make run`
- Ensure everything works :-)
